build/device/usb/__init__.py

```diff
@@ -1,17 +1,32 @@
-# Copyright (C) 2009-2017 Wander Lairson Costa
```

Comment:
We would rather update pyUSB with one of the released version: https://github.com/pyusb/pyusb/tags

Reply:
Indeed, good idea